src/frontend/jest.config.js

```diff
@@ -12,4 +12,5 @@ module.exports = {
   testEnvironment: 'jsdom',
   testMatch: [__dirname + '/**/*.spec.+(ts|tsx|js)'],
   testURL: 'https://localhost',
+  testTimeout: 200000,
```

Comment:
I was still facing random timeout on tests with `toMatchImageSnapshot` when running all tests

Reply:
It's look like an issue with your setup. There is no timeout problem with the CI

Reply:
well I'm on longer facing this issue, we can wait to see if this occur in the CI to update the conf